addons/azureSets/azuresets.lua

```diff
@@ -234,7 +251,11 @@ windower.register_event('addon command', function(...)
 
         elseif comm == 'spellset' or comm == 'set' then
             if args[1] ~= nil then
-                set_spells(args[1])
+                if args[2] ~= nil then
+                    set_spells(args[1], args[2])
+                else
+                    set_spells(args[1])
+                end
```

Comment:
`set_spells(args[1])` is the same as `set_spells(args[1], nil)`, so the nil check is kind of redundant here.